source/deploy/mattermost-changelog.md

```diff
@@ -31,6 +31,7 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
   - Original 9.2.0 release
 
 ### Important Upgrade Notes
+ - Fixed data retention policies to run jobs when any custom retention policy is enabled even when the global retention policy is set to **keep-forever**. Before this fix, the enabled custom data retention policies wouldn’t run as long as the global data retention policy was set to **keep-forever** or was disabled. After the fix, the custom data retention policies will run automatically even when the global data retention policy is set to **keep-forever**. Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom data retention policies before upgrading, and then re-enable them again after upgrading.
  - Please read the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) before upgrading.
```

Comment:
A bit of a late review: Given that the important upgrade notes for this release are already here, this sentence needs to be more accurate as it suggests that there are more notices for this release. Can we clarify that the link contains only additional notes for other releases?

Reply:
I'll make this edit.

Reply:
Thanks @amyblais